terraform/deploy/app/frontendservice.tf

```diff
@@ -25,11 +25,31 @@ module "ecs-fargate-task-definition" {
   environment = [
     {
       name  = "REACT_APP_UI_LOGIN_URL"
-      value = "https://placeholder-url" # to be amended at a later date (in another ticket)
+      value = "https://${module.ecs-fargate-service.fqdn}"
```

Comment:
I don't think this is needed anymore as Amplify is handling the auth?

Reply:
@IanUtley ?

Reply:
Actually just checked and you're right it's not mentioned anywhere else - will remove.